error_middleware.go

```diff
@@ -58,8 +58,11 @@ func writeError(log logrus.FieldLogger, w negroni.ResponseWriter, err error) {
 	}
 
 	isCauseValidationErrors := errors.IsRootCause(err, &errors.ValidationErrors{})
+	isCausePaginationBadRequest := errors.IsRootCause(err, &BadRequestError{})
```

Comment:
I find this line kind of wrong. You name the variable `isCausePaginationBadRequest` but the root cause error you get is a `BadRequestError`. So there is no notion of pagination in the `BadRequestError` error type. Hence I think there is a naming problem somewhere.

Reply:
Yes definitely !